src/interfaces/options/integer.options.ts

```diff
@@ -1,4 +1,4 @@
 export interface IntegerOptions {
-    maximum?: number,
-    minimum?: number,
+    maxValue?: number,
```

Comment:
I renamed these options to be consistent with `minItems/minLength`. I'm a little torn here because the `minimum/maximum` naming is what OpenAPI uses, but I actually got a bit confused during this PR and found that using `minFoo` was clearer.

 - `minLength` => string length
 - `minItems` => array size
 - `minValue` => numeric magnitude